### PR TITLE
Lev 177 node upgrade

### PR DESCRIPTION
**Pull request by nBaqai**
Upgraded node version and removed vulnerabilities

### CODE REVIEW

**Review by mehar-chauhan — APPROVED** [review]

**Comment by mehar-chauhan** [comment]
LGTM, but you should await for @aarongill1 's update before proceeding

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine3.15 as base
+FROM node:20-alpine as base
```
Comment (aarongill1) [inline Dockerfile:1]: Might want to add a specific version here but not necessary - leave it to your decision

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
   },
   "devDependencies": {
     "axe-core": "^4.5.2",
-    "cypress": "^12.17.1 ",
+    "cypress": "^12.17.1",
```
Comment (aarongill1) [inline package.json:49]: Remove the "^" here - if another version of Cypress is installed there's a version mis-match with the Docker-e2e Dockerfile and it causes issues.

Comment (nBaqai) [reply package.json]: Cool, just made changes to remove it

File: Dockerfile (hunk 1 above)
Comment (nBaqai) [reply Dockerfile]: Added 20-alpine3.18 version

**Review by aarongill1 — APPROVED** [review]

**Pull request merged**